### PR TITLE
add spiderpool metrics markdown

### PR DESCRIPTION
**Pull request by Icarus9913**
Signed-off-by: Icarus9913 <icaruswu66@qq.com>

### CODE REVIEW

File: pkg/metric/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Metrics
+
+Spiderpool can be configured to serve [Opentelemetry](https://opentelemetry.io/) metrics.
+Spiderpool metrics provide the insight of Spiderpool Agent and Spiderpool Controller.
+
+## Get Started
+
+### Enable Metric support
+
+Firstly, please ensure you have installed the spiderpool and configure the CNI file, refer [install](./install.md) for details
```
Comment (windsonsea) [inline pkg/metric/README.md:10]:
```suggestion
Firstly, please ensure you have installed the spiderpool and configure the CNI file, refer to [install](./install.md) for details
```

File: pkg/metric/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Check daemonset `spiderpool-agent` environment `SPIDERPOOL_ENABLED_METRIC` whether is already set to `true` or not.
```
Comment (windsonsea) [inline pkg/metric/README.md:12]:
```suggestion
Check the environment variable `SPIDERPOOL_ENABLED_METRIC` of the daemonset `spiderpool-agent`  for whether it is already set to `true` or not.
```

File: pkg/metric/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Check deployment `spiderpool-controller` environment `SPIDERPOOL_ENABLED_METRIC` whether is already set to `true` or not.
```
Comment (windsonsea) [inline pkg/metric/README.md:14]:
```suggestion
Check the environment variable `SPIDERPOOL_ENABLED_METRIC` of the deployment `spiderpool-controller` for whether it is already set to `true` or not.
```

File: pkg/metric/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+```shell
+kubectl -n kube-system get daemonset spiderpool-agent -o yaml
+------
+kubectl -n kube-system get deployment spiderpool-controller -o yaml
+```
+
+You can set one of them or both of them to `true`.
```
Comment (windsonsea) [inline pkg/metric/README.md:22]:
```suggestion
You can set one or both of them to `true`.
```

File: pkg/metric/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+For example, let's enable spiderpool agent metrics, just execute `helm upgrade --set spiderpoolAgent.prometheus.enabled=true`.
```
Comment (windsonsea) [inline pkg/metric/README.md:23]:
```suggestion
For example, let's enable spiderpool agent metrics by running `helm upgrade --set spiderpoolAgent.prometheus.enabled=true`.
```

File: pkg/metric/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+## Metric reference
+
+### Spiderpool Agent
+
+Spiderpool agent exports some metrics related with IPAM allocation and deallocation. Currently, those include:
+
+
+| Name                                          | description                                                                                                                    |
+|-----------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| ipam_allocation_total_counts                  | Count of the number of Spiderpool Agent received the IPAM allocation requests, prometheus type: counter                        |
```
Comment (windsonsea) [inline pkg/metric/README.md:34]:
```suggestion
| ipam_allocation_total_counts                  | Count of the number of Spiderpool Agents that received the IPAM allocation requests, prometheus type: counter                        |
```

File: pkg/metric/README.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+| ipam_allocation_failure_counts                | Number of Spiderpool Agent IPAM allocation failures, prometheus type: counter                                                  |
+| ipam_allocation_average_duration_seconds      | The average time duration of total Spiderpool Agent allocation process, prometheus type: gauge                                 |
```
Comment (windsonsea) [inline pkg/metric/README.md:36]:
```suggestion
| ipam_allocation_average_duration_seconds      | The average duration of all Spiderpool Agent allocation processes, prometheus type: gauge                                 |
```

File: pkg/metric/README.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+| ipam_allocation_max_duration_seconds          | The maximum time duration of Spiderpool Agent allocation process (per-process), prometheus type: gauge                         |
```
Comment (windsonsea) [inline pkg/metric/README.md:37]:
```suggestion
| ipam_allocation_max_duration_seconds          | The maximum duration of Spiderpool Agent allocation process (per-process), prometheus type: gauge                         |
```
Need to replace `time duration` with `duration`.

**Review by weizhoublue — DISMISSED** [review]

**Review by weizhoublue — APPROVED** [review]

**Review by windsonsea — APPROVED** [review]

**Pull request merged**